domino/xsp/domino-faces-config.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like this could be the answer http://stackoverflow.com/questions/8386543/copying-file-from-one-project-to-another-in-maven. I'll give it a go.